plugin/bind/setup_test.go

```diff
@@ -45,3 +45,39 @@ func TestSetup(t *testing.T) {
 		}
 	}
 }
+
+func Test_isIn(t *testing.T) {
```

Comment:
this is admirable, but can we rather remove the function `isIn` and rather replace it with `slices.Contains(except, ip)` ?